repo/fsrepo/datastores.go

```diff
@@ -382,6 +383,17 @@ func BadgerdsDatastoreConfig(params map[string]interface{}) (DatastoreConfig, er
 		}
 	}
 
+	truncate, ok := params["truncate"]
```

Comment:
How about (could do this with `syncWrites` above too):
```
c.truncate, ok = params["truncate"].(bool)
if !ok {
    c.truncate = true
}
```

Reply:
That is much cleaner (I didn't even knew I could perform those two operations in the same line) but it will silently fail for cases when the user puts the value as a string, e.g., `"syncWrites": "false"` will actually be interpreted as `true` without the user even noticing it.